.github/workflows/ci.yml

```diff
@@ -21,6 +21,8 @@ jobs:
     - uses: actions/checkout@v2
     - name: Setup .NET
       uses: actions/setup-dotnet@v1
+      with:
+        include-prerelease: true
```

Comment:
I think this section may not be necessary as I just had to update my global.json and it was picked up automatically.